content/zh-cn/blog/contributor1.md

```
@@ -0,0 +1,35 @@
+### <font style="color:rgb(31, 35, 40);">commits 记录/PR 贡献等：</font>
+[<font style="color:rgb(31, 35, 40);">https://github.com/sofastack/sofa-serverless/pulls?q=is%3Apr+is%3Aclosed+author%3Aqixiaobo</font>](https://github.com/sofastack/sofa-serverless/pulls?q=is%3Apr+is%3Aclosed+author%3Aqixiaobo)<font style="color:rgb(31, 35, 40);">  
+</font>
+
+![](https://intranetproxy.alipay.com/skylark/lark/0/2024/png/149473/1727419260641-0f08d99d-2726-4af0-9534-eb659d7c8527.png)
+
+[<font style="color:rgb(31, 35, 40);">https://github.com/sofastack/sofa-ark/pulls?q=is%3Apr+is%3Aclosed+author%3Aqixiaobo</font>](https://github.com/sofastack/sofa-ark/pulls?q=is%3Apr+is%3Aclosed+author%3Aqixiaobo)<font style="color:rgb(31, 35, 40);">  
+</font>![](https://intranetproxy.alipay.com/skylark/lark/0/2024/png/149473/1727418763098-5ec46423-4d35-4075-b642-4178151d94c1.png)
+
+### <font style="color:rgb(31, 35, 40);">突出贡献：</font>
+<font style="color:rgb(31, 35, 40);">扩展了 springboot 1.x 的支持，沉淀了10+类组件在落地多应用模式的最佳实践和治理方案，并深度参与如何从框架角度定义标准治理方案，完成了 dubbo 2.6 和 apollo 的 adapter 适配组件。  
+</font>![](https://intranetproxy.alipay.com/skylark/lark/0/2024/png/149473/1727418765065-e4c961c3-3f07-462b-ac95-7316a21462ed.png)
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)